conda error when installing deps from environment.yaml

I added an environment.yml (see below) to install a few extra python packages. Unfortunately, when after pushing a tag the workflow fails with the error (see [here](https://github.com/SyneRBI/PETRIC-MaGeZ/actions/runs/10635488493/job/29485184326#step:4:52)):

```
CondaValueError: could not parse 'name: PETRIC-MAGEZ' in: environment.yml
```

My environment.yml contains
```
name: PETRIC-MAGEZ
channels: [conda-forge]
dependencies:
- my_package_1
- my_package_2
```

I guess I don't understand what the name of the conda env should be.

still failing :( - now in the channels section 

https://github.com/SyneRBI/PETRIC-MaGeZ/actions/runs/10635744880/job/29486030531

```
CondaValueError: could not parse '- conda-forge' in: environment.yml
Error: Process completed with exit code 1.
```

--- a/.github/workflows/run.yml
+++ b/.github/workflows/run.yml
@@ -22,7 +22,8 @@ jobs:
           xargs -a apt.txt sudo apt-get install -y
         fi
         if test -f environment.yml; then
-          conda install --file environment.yml
+          sed -i '/^name:.*/d' environment.yml
+          conda env update
         fi
         if test -f requirements.txt; then
           pip install -r requirements.txt

